### PR TITLE
Yarn.lock and documentation changes

### PR DESCRIPTION
**Pull request by Rhys362**
Hi @iGitScor 

The yarn.lock file is now added and the documentation updated. Please let me know if everything is ok :+1: 

Issues #5 #9

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -2,8 +2,16 @@
...
+
+```sh
+$ npm install -g kickstarter-tools
+```
+
+Using yarn
+
 ```sh
-$ npm install -g {Kickstarter repo}
+$ yarn add -g kickstarter-tools
```
Comment (iGitScor) [inline docs/getting-started.md]: I didn't find `-g` shortcut anywhere.

[Documentation](https://yarnpkg.com/fr/docs/cli/global) for global command

**Comment by iGitScor** [comment]
Hi @Rhys362,

I didn't mention it before, sorry, but could you try to use yarn in the travis build ?
[Here](https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn) some documentation on the topic.

Thank you

**Comment by Rhys362** [comment]
Hi @iGitScor 

Done and done: https://travis-ci.org/Rhys362/kickstarter/builds/207772029

**Comment by iGitScor** [comment]
Hi @Rhys362,

it's perfect.

Close #5
Close #9 too

**Comment by Rhys362** [comment]
:smile:

**Comment by iGitScor** [comment]
@Rhys362, Do you want to work on another project ? We don't have many improvement for this, for now.

**Comment by Rhys362** [comment]
@iGitScor, Yes please!

**Pull request merged**